packages/@expo/metro-config/src/transform-worker/__tests__/metro-transform-worker.test.ts

```diff
@@ -8,12 +8,43 @@
  * Fork of the Metro transformer worker, but with additional transforms moved to `babel-preset-expo` and modifications made for web support.
  * https://github.com/facebook/metro/blob/412771475c540b6f85d75d9dcd5a39a6e0753582/packages/metro-transform-worker/src/index.js#L1
  */
+
+import {
+  TraceMap,
+  EncodedSourceMap,
+  originalPositionFor,
+  generatedPositionFor,
+} from '@jridgewell/trace-mapping';
 import { Buffer } from 'buffer';
 import * as fs from 'fs';
 import { vol } from 'memfs';
-import { JsTransformerConfig, JsTransformOptions } from 'metro-transform-worker';
+import { fromRawMappings } from 'metro-source-map';
+import type { JsTransformerConfig, JsTransformOptions, JsOutput } from 'metro-transform-worker';
 import * as path from 'path';
 
+// NOTE(kitten): metro-source-map does not provide typings for its exported functions
+declare module 'metro-source-map' {
```

Comment:
Move this here https://github.com/expo/expo/blob/main/packages/@expo/cli/ts-declarations/metro-source-map/index.d.ts

Reply:
Small side-note, if these are supposed to be global, I'm vaguely sure that the root `tsconfig.json` doesn't include them correctly 🤔 That's probably why I missed them, since I open my editor from the `expo` repo root folder.